sql.js

```diff
@@ -216,563 +216,8 @@ module.exports = {
         BEGIN
             CREATE USER [${user}] FOR LOGIN [${user}]
         END
-        EXEC sp_addrolemember 'db_owner', '${user}'`;
-    },
-    enableDatabaseDiagrams: function(name) {
```

Comment:
This is resolved in the commit of merge master to the feat branch. I think this is happened due to uis-2550 is not having the latest code. Now I merge master to the feat branch, It is resolved